client/app/bundles/HelloWorld/reducers/person.jsx

```diff
@@ -0,0 +1,25 @@
+import { combineReducers } from 'redux-immutable';
+import actionTypes from '../constants/helloWorldConstants';
+
+// Revision 161029:
+// the helloWorldstore wrapper doesn't solve the issue with Immutable.js.
+// what if we have things like age, email besides name? although in this
+// simple example app it's OK to deal with an Immutable object with
+// only one reducer but people will soon run into trouble when we want
+// to seperate reducers and use combineReducers to produce the Immutable
+// object because the original combineReducers expects plain object
+// I think it's better to integrate redux-immutable otherwise they will be
+// stuck on this
```

Comment:
@robwise @justin808 
I see. However when I first saw this example, due to the use of Immutable.js in it, I thought it was about teaching people what they may need do in their future development rather than giving a simplest example. Without it, the example can be much simpler and I don't think Immutable.js is an necessity to everyone. Anyway, with it, it's better to tell people about reselect, redux-immutable somewhere in the doc if it's not in this hello world example.